core/ecschema-editing/src/Editing/Entities.ts

```diff
@@ -196,15 +196,18 @@ export class Entities extends ECClasses {
 
     const baseClassSchema = !baseClassKey.schemaKey.matches(entityKey.schemaKey) ? await this._schemaEditor.getSchema(baseClassKey.schemaKey) : entity.schema;
     if (baseClassSchema === undefined) {
-      return { errorMessage: `Schema Key ${baseClassKey.schemaKey.toString(true)} not found in context` };
```

Comment:
The error handling changes you made are fine for now. This will all be changing soon anyway.